src/store/modules/stock/index.ts

```diff
@@ -0,0 +1,18 @@
+import actions from './actions'
+import getters from './getters'
+import mutations from './mutations'
+import { Module } from 'vuex'
+import StockState from './stockState'
```

Comment:
Rename file to follow PascalCase pattern
```suggestion
import StockState from './StockState'
```